src/odemis/util/spot.py

```diff
@@ -132,6 +133,15 @@ def SpotIntensity(data, background=None):
 
 
 def FindCenterCoordinates(image, smoothing=True):
+    warnings.warn(
+        "FindCenterCoordinates will become deprecated, use RadialSymmetryCenter instead.",
+        PendingDeprecationWarning
+    )
+    jc, ic = RadialSymmetryCenter(image, smoothing)
+    return ic, jc
+
```

Comment:
You're now causing all SECOM/DELPHI code to generate a warning.
I'd rather have all the callers be updated.